Makefile

```diff
@@ -284,6 +284,10 @@ install-sdk:
 	mkdir -p "$(DESTDIR)$(prefix)"/lib
 	cp -R "$(CURDIR)"/lib/* "$(DESTDIR)$(prefix)"/lib
 	find "$(DESTDIR)$(prefix)"/lib -type f -exec chmod 644 {} \;
+	mkdir -p "$(DESTDIR)$(prefix)"/default
```

Comment:
I think I'd prefer packages-default for this to group more nicely with packages/.packages.